metricbeat/module/php_fpm/process/process.go

```diff
@@ -0,0 +1,65 @@
+package process
```

Comment:
You will need to add headers here, you can run `make add-headers` and they are be automatically added to all files.